web-console/knikubevirt/cluster-dashboard/cluster-dashboard.md

```diff
@@ -0,0 +1,71 @@
+# Cluster Overview Dashboard
+
+The Cluster Overview dashboard provides a birds-eye view of the cluster's current health, capacity, inventory, utilization, and activity.
+
+The Overview dashboard is the first cluster-level dashboard that appears when the Dashboards nav item is selected. Additional cluster-level dashboards are available as tabs.
+
+![Cluster Overview Dashboard Healthy](img/1-0-cluster-healthy.png)
+
+The dashboard is made of 7 cards, each of which was designed to be:
+
+1. Contextual - able to be used across products
+2. Reusable - applicable to multiple dashboards
+3. Responsive - viewable on any screen size
+
+![Cluster Overview Dashboard Unhealthy](img/1-1-cluster-unhealthy.png)
+
+The Overview dashboard is a particularly useful place to identify cluster-wide problems and begin troubleshooting alerts.
+
+## Cards
+
+### Details card
+
+![Details card](img/0-1-details.png)
```

Comment:
Should this stay focused on just the Cluster Overview Card for this PR?

Reply:
Looks more boring, but I suppose so. Change incoming.

Reply:
Heh, just nice and clear now! :)